src/main/java/com/endava/soa4/PropertyLoader.java

```diff
@@ -0,0 +1,13 @@
+package com.endava.soa4;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyLoader {
+    public static void main(String[] args) throws IOException {
```

Comment:
Please make it as a simple methot instead ov publis static void main. And you will call it from DriverManager class.

Reply:
Done